torch/csrc/distributed/rpc/request_callback.cpp

```diff
@@ -9,13 +9,13 @@ namespace rpc {
 
 using namespace torch::distributed::autograd;
 
-std::shared_ptr<JitFuture> RequestCallback::operator()(Message& request) const {
+std::shared_ptr<JitFuture> RequestCallback::operator()(Message& request, std::shared_ptr<LazyStreamContext> ctx) const {
```

Comment:
Let's break into long line into shorter pieces. Can you try run `clang-format -i` on these files.

Reply:
addressed